server_admin/topics/authentication/webauthn.adoc

```diff
@@ -167,27 +166,61 @@ A new user carries out the following operations :
 
 Existing user::
 
-When existing users try to log in, they are required to register their WebAuthn authenticator automatically :
+In case that `WebAuthn Authenticator` is setup as required as shown in the first example, then when existing users try to log in,
+they are required to register their WebAuthn authenticator automatically :
 
 - Open the login form.
 - Fill in items, click `Save` and  click `Login`.
 - When the users log in, they are required to register their WebAuthn authenticator.
 - After successful registration, the user's browser asks the user to enter the text as their just registered WebAuthn authenticator's label.
 
-===== View Registered WebAuthn Authenticator
+==== Passwordless WebAuthn
+
+WebAuthn is often used for two-factor authentication, however it can be desired to use it also as first factor authentication. In this case,
+user with `passwordless` WebAuthn credential will be able to authenticate to {project_name} without password. {project_name} allows to use WebAuthn
+as both passwordless and two-factor authentication mechanism in the context of single realm and even in the context of single authentication flow.
+
+Administrator may typically require that Security Keys registered by users for the WebAuthn passwordless authentication must meet different
+(usually stronger) requirements. For example those security keys may require users to authenticate to that security key using PIN, or the
+security key should be attested with stronger certificate authority.
+
+Because of this, {project_name} allows administrator to configure separate `WebAuthn Passwordless Policy`. There is also separate required action
```

Comment:
add "a" before "also"

Reply:
Hmmm... I am not good at english, but sentence "There is a also separate required action.." sounds really strange?

Did you meant:
add "a" INSTEAD OF "also" ? 

So the sentence will be like "There is a separate required action ..." .

Reply:
Maybe `There is also a separate required action...` was meant by @andymunro ?

Reply:
@iankko ah, I think that makes sense :) I've changed this sentence already to `There is a separate required action...` . Is it ok to keep it that way or should I change it to `There is also a separate required action..` . WDYT @andymunro @iankko ?

Reply:
@mposolda `There is a separate required action ....` looks good enough to me.

@andymunro Kind request - could you please review this one with priority & point out the points, that still need to be addressed we to be able to merge this one? People are already asking for this change (see the related code PR for comments).

TYVM! Jan